master-firmware/src/strategy.cpp

```diff
@@ -386,6 +386,29 @@ void strat_push_the_bee_v2(point_t start, float bee_height, float forward_motion
     trajectory_wait_for_end(TRAJ_FLAGS_SHORT_DISTANCE);
 }
 
+bool strat_lever_is_full(enum lever_side_t lever_side)
+{
+    bool full = true;
+
+    if (lever_side == LEVER_SIDE_LEFT) {
+        messagebus_topic_t* topic = messagebus_find_topic_blocking(&bus, "/lever/left");
```

Comment:
Probably safer to use `messagebus_find_topic` (nonblocking) and handle the case where the topic does not exist. Just in case someone disables something or changes a topic name or whatever.

Also, this function could definitely use some refactoring, but given that this is in strategy... who cares.

Reply:
Refactoring? Ain't nobody got time for that